lib/fileio/commands/baseExportCommand.cpp

```diff
@@ -0,0 +1,328 @@
+//
+// Copyright 2016 Pixar
+// Copyright 2020 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "baseExportCommand.h"
+
+#include <vector>
+
+#include <pxr/pxr.h>
+
+#include <maya/MGlobal.h>
+#include <maya/MSyntax.h>
+#include <maya/MArgDatabase.h>
+#include <maya/MArgList.h>
+#include <maya/MFileObject.h>
+#include <maya/MSelectionList.h>
+
+#include <mayaUsd/fileio/jobs/jobArgs.h>
+#include <mayaUsd/fileio/utils/writeUtil.h>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+namespace {
+
+} // namespace
+
+MAYAUSD_NS_DEF {
+
+MayaUSDExportCommand::MayaUSDExportCommand()
+{
+}
+
+MayaUSDExportCommand::~MayaUSDExportCommand()
+{
+}
+
+MSyntax MayaUSDExportCommand::createSyntax()
+{
+    MSyntax syntax;
+
+    // These flags correspond to entries in
+    // UsdMayaJobExportArgs::GetDefaultDictionary.
+    syntax.addFlag(kMergeTransformAndShapeFlag,
```

Comment:
A minor change, but all short names are now identifiers like this instead of hard-coded strings like were in the original Pixar source.